### ISSUE TITLE
`--incompatible_check_sharding_support` can cause intermittent test failures

### ISSUE THREAD

**Issue by fmeum** [bazelbuild/bazel#22028]
### Description of the bug:

```
/workdir/src/test/shell/integration/BUILD:123:8: Testing //src/test/shell/integration:loading_phase_tests (shard 1 of 8) failed: Sharding requested, but the test runner did not advertise support for it by touching TEST_SHARD_STATUS_FILE. Either remove the 'shard_count' attribute, use a test runner that supports sharding or temporarily disable this check via --noincompatible_check_sharding_support.
```

seen on
https://buildkite.com/bazel/bazel-bazel-github-presubmit/builds/20155#018db413-588a-46b0-94c2-7b49eb7a7b8c/49-916

### Which category does this issue belong to?

Core

**Comment by fmeum** [comment]
Good idea, let's try that first and see whether any failures remain with it: https://github.com/bazelbuild/bazel/pull/22098

**Comment by iancha1992** [comment]
A fix for this issue has been included in [Bazel 7.2.0 RC1](https://github.com/bazelbuild/bazel/releases/tag/7.2.0rc1). Please test out the release candidate and report any issues as soon as possible.
If you're using Bazelisk, you can point to the latest RC by setting USE_BAZEL_VERSION=7.2.0rc1. Thanks!
